app/iht/utils/CommonHelper.scala

```diff
@@ -70,6 +70,23 @@ object CommonHelper {
     fmt.format(n)
   }
 
+  def formatCurrencyForInput(n: String): Any = {
+    if(!Option(n).getOrElse("").isEmpty) {
+      try{
+        val b = BigDecimal(n)
+        val fmt = new java.text.DecimalFormat("#####.##")
+        fmt.setDecimalSeparatorAlwaysShown(false)
+        fmt.setMinimumFractionDigits(2)
+        fmt.format(b)
+      } catch {
+        case _: Throwable => n
```

Comment:
Make this a more specific exception if possible so we don't lose real exceptions.